custom_components/tuya_local/devices/calex_convector_heater.yaml

```diff
@@ -0,0 +1,159 @@
+name: Calex Smart Convector Heater
+products:
+  - id: yqnv4v88805rpkr1
+    name: Calex Smart Heater
+primary_entity:
+  entity: climate
+  dps:
+    - id: 1
+      name: hvac_mode
+      type: boolean
+      mapping:
+        - dps_val: true
+          value: "heat"
+          icon: "mdi:radiator"
+        - dps_val: false
+          value: "off"
+          icon: "mdi:radiator-disabled"
+    - id: 2
+      name: temperature
+      type: integer
+      unit: ℃
+      range:
+        min: 5
+        max: 35
+    - id: 3
+      name: current_temperature
+      readonly: true
+      type: integer
+      unit: ℃
+      range:
+        min: -20
+        max: 50
+    - id: 101
+      name: preset_mode
+      type: string
+      mapping:
+        - dps_val: H1
+          value: eco
+        - dps_val: H2
+          value: boost
+        - dps_val: AF
+          value: away
+        - dps_val: P7
+          value: activity
+secondary_entities:
+  - entity: lock
+    name: Child lock
+    category: config
+    dps:
+      - id: 7
+        type: boolean
+        name: lock
+        mapping:
+          - dps_val: true
+            icon: "mdi:hand-back-right-off"
+          - dps_val: false
+            icon: "mdi:hand-back-right"
+  - entity: sensor
```

Comment:
This can be converted to hvac_action on the climate entity, with values "off", "idle" and "heating".  

```
   - id: 11
     name: hvac_action
     type: string
     mapping:
       - constraint: hvac_mode
          conditions:
            - dps_val: false
              value: "off"
            - dps_val: true
              mapping:
                - dps_val: standby
                  value: idle
                - dps_val: working
                  value: heating
```

Reply:
Good point, moved to hvac_action